src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -0,0 +1,337 @@
+* DOM mutation tracking: Once loaded, session replay monitors DOM changes and reports them as compressed payloads. The size of these payloads correlates with the complexity of the DOM tree.
+* Performance optimization techniques: Payload size can also increase if the DOM is large. However, session replay uses a number of techniques to reduce payload size, such as:
+    * GZIP compression
+    * Duplication reduction
+    * Strategic payload harvesting: It only sends a large (compressed) payload when the full DOM is captured, which typically only happens when the replay feature is first imported. After that, it will only send small payloads that relate to the end-user's actions.
+</Collapser>
+
+<Collapser
+    id="data-retention-limits"
+    title="What are the data retention limits?"
+>
+    Replays have a standard retention period of 8 days.
+</Collapser>
+
+</CollapserGroup>
+
+
+## Get started [#get-started]
+
+<Steps>
+
+  <Step>
+    ### Enable and configure session replay [#enable-configure-settings]
+
+    1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > All Capabilities > Browser**.
+    2. Select your browser app.
+    3. In the left-hand menu, click  **Application settings**.
+    2. On the Application settings page, make sure **Pro** or **Pro + SPA** browser agent is selected. Session replay is not available for the Lite browser agent.
+    3. Toggle ON **Session tracking** and **Session replay** settings.
+    </Step>
+
+  <Step>
+    ### Configure session sampling rates [#configure-sampling-rates]
+    
+    Make selections for the following sampling rates:
+
+    * **Session sampling rate** records a random sample of all user sessions.
+
+    * **Errored session sample rate** records a random sample of all errored sessions. The agent buffers for up to 30 seconds on the client-side, and if errors occur, the session will be recorded. If an error does not occur, the replay is not harvested. 
+
+For example, if you set the session sampling rate to 50% and the error sampling rate to 100%, it means that:
+
+* Half of all user sessions will be recorded.
+* All errored sessions will be recorded, even if they are not part of the 50% sample.
+
+  </Step>
+
+    <Step>
+    ### Configure privacy settings [#configure-privacy-settings]
+
+    New Relic takes your data privacy seriously. The default setting masks everything by replacing all text with asterisks. 
+
+    Here's an example of how a replay looks with the default privacy settings:
+
+    <img
+        title="Example of default privacy settings"
+        alt="Example of default privacy settings"
+        src={browserTextMaskingExample}
+    />
+
+If you need to change the default privacy settings, you can choose specific classes and attributes to mask, block, or ignore:
+
+    * **Mask** means specific text will be replaced with asterisks (*). For example, you could mask sensitive user input such as login information.
```

Comment:
```suggestion
**Mask** means specific text will be replaced with asterisks (*). For example, you could mask sensitive user input such as login information.
```

Reply:
I'm not a fan of styling these as bullets. I think just removing the bullet will help, but you can also explore making these h3s, using tabs, side-by-sides, etc.